scripts/actions/mobskills/Proboscis.lua

```diff
@@ -0,0 +1,33 @@
+-----------------------------------
+-- Proboscis
+-- Description: Drains MP and steals 1 Buff
+-- Type: Magical (Element)
+-- Range: Conal
+-- Utsusemi/Blink absorb: 1 shadow
+-----------------------------------
+---@type TMobSkill
+local mobskillObject = {}
+
+mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    -- Dark magical damage calculation
+    local dmgmod = 1.5
+    local info = xi.mobskills.mobMagicalMove(mob, target, skill, mob:getWeaponDmg() * dmgmod, xi.element.DARK, dmgmod, xi.mobskills.magicalTpBonus.MAB_BONUS, 1)
```

Comment:
mobMagicalMove returns integer, not a table.  So in this case, it is the amount of damage

Reply:
In addition, it looks like this file was renamed as well.  All skill/spell scripts should be lowercase filenames